common/views/components/GlobalInfoBarContext/GlobalInfoBarContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { createContext, FunctionComponent, ReactNode, useState } from 'react';
+
+type Props = {
+  children: ReactNode;
+};
+type ContextProps = {
+  isVisible: boolean;
+  setIsVisible: (isVisible: boolean) => void;
+};
+const GlobalInfoBarContext = createContext<ContextProps>({
+  isVisible: false,
+  setIsVisible: () => {
+    // noop
```

Comment:
not setting visible ?

Reply:
Do you mean the naming? The common pattern for the `useState` hook us `[value, setValue]`, [as you can see here](https://reactjs.org/docs/hooks-state.html). We tend to prefix `boolean`s with `is` e.g: `isCondition`.

Does this answer your question?

Reply:
ahh, its the default values which was confusing me. cause the context props wants a boolean but the setIsVisible doesnt expect anything.